### PR TITLE
doc/metrics: stop tuning job's scrape_interval now that results are c…

### PR DESCRIPTION
**Pull request by simondeziel**
…ached 8s only

Signed-off-by: Simon Deziel <simon.deziel@canonical.com>

### CODE REVIEW

File: doc/metrics.md [hunk 1]
```diff
@@ -2,7 +2,8 @@
 LXD provides metrics for all running instances. Those covers CPU, memory, network, disk and process usage and are meant to be consumed by Prometheus and likely graphed in Grafana.
 In cluster environments, LXD will only return the values for instances running on the server being accessed. It's expected that each cluster member will be scraped separately.
 The instance metrics are updated when calling the `/1.0/metrics` endpoint.
-They are cached for 15s to handle multiple scrapers. Fetching metrics is a relatively expensive operation for LXD to perform so we would recommend scraping at a 30s or 60s rate to limit impact.
+They are cached for 8s to handle multiple scrapers. Fetching metrics is a relatively expensive operation for LXD to perform so consider scraping at a higher than default interval (15s)
+if the impact is too high.
```
Comment (tomponline) [inline doc/metrics.md:6]: Is it 10s by default?

Comment (tomponline) [reply doc/metrics.md:6]: Ah it is apparently 15s, but some sources sayv10s, maybe it changed.

Comment (simondeziel) [reply doc/metrics.md:6]: Actually, the default is 60s according to the [upstream doc](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config). However, the [prometheus snap](https://snapcraft.io/prometheus) creates prometheus.yml by copying it from the upstream [config example](https://github.com/prometheus/prometheus/blob/main/documentation/examples/prometheus.yml) that overrides it to 15s. I think I should simply remove the 15s mention, what do you think @tomponline?

According to https://www.robustperception.io/keep-it-simple-scrape_interval-id, 10-60s is good and it's best to only use one scrape_interval throughout all scrape jobs.

Comment (tomponline) [reply doc/metrics.md:6]: It was this page that gave the impression the default was 15s but its just an example https://prometheus.io/docs/prometheus/latest/getting_started/

Happy to leave it out of our examples though

**Pull request merged**